components/Users.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+
+const Users = ({ userId, userName }) =>
```

Comment:
If it's rendering one user, the name of the component should be singular, something like `User`, but we can do better than just "User". Let's use `UserCard` or anything more specific.